api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -----------
+    Definitions
+    -----------
+    Nakadi at its core aims at being a generic and content-agnostic event broker with a convenient API.
+    In doing this, Nakadi abstracts away, as much as possible, details of the backing messaging
+    infrastructure. The single currently supported messaging infrastructure is Kafka (Kinesis is planned
+    for the future).
+
+    In Nakadi every Event has an EventType, and a **stream** of Events is exposed for each registered
```

Comment:
Do you have any concept of forwarding/duplicating some percentage of event-type traffic to another stream so that we can build independent staging infrastracture? Or you will push idea to have another consumer for this purpose?